infrastructure/src/main/java/org/corfudb/infrastructure/ManagementServer.java

```diff
@@ -249,6 +255,22 @@ boolean checkBootstrap(CorfuMsg msg, ChannelHandlerContext ctx, IServerRouter r)
     }
 
     /**
+     * Forward an orchestrator request to the orchestrator service.
+     *
+     * @param msg corfu message containing ORCHESTRATOR_REQUEST
+     * @param ctx netty ChannelHandlerContext
+     * @param r   server router
+     */
+    @ServerHandler(type = CorfuMsgType.ORCHESTRATOR_REQUEST, opTimer = metricsPrefix
+            + "orchestrator-request")
+    public synchronized void handleOrchestratorMsg(CorfuPayloadMsg<OrchestratorRequest> msg,
```

Comment:
Please annotate with javax.annotation.* annotations

Reply:
You mean Nonnull on the server's handlers ?

Reply:
Everywhere, please.

Reply:
done.